DEHPCommon.Tests/UserPreferenceHandler/expectedUserPreference.json

```diff
@@ -0,0 +1,6 @@
+﻿{
```

Comment:
You do not need to prpovide a default file. An empty should be created automatically if it does not exist.

Reply:
This file was a sample for testing, to check that is write method can create proper Json: Assert.AreEqual(expectedUserPreferenceContent, writtenContent). 
Should I leave it or delete and change testing method?